modules/core/test/image/testImageMorphology.cpp

```diff
@@ -137,9 +164,14 @@ TEST_CASE("Binary image morphology", "[image_morphology]")
           1, 1, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 1};
       vpImage<unsigned char> I_dilatation_ref(image_data_dilatation, 8, 16, true);
       CHECK((I_dilatation_ref == I_dilatation));
+      #if defined(VISP_BUILD_DEPRECATED_FUNCTIONS)
+      vpImage<unsigned char> I_deprecated_dilatation = I;
+      vpImageMorphology::dilatation(I_deprecated_dilatation, connexity);
+      CHECK((I_dilatation_ref == I_deprecated_dilatation));
+      #endif
 
```

Comment:
Here also and in other lines below

Reply:
Done in commit af0e02632. Thanks for the help =)